app/src/main/res/layout/onlinefeedview_activity.xml

```diff
@@ -64,103 +66,67 @@
-                    android:layout_marginStart="8dp"
-                    android:layout_marginEnd="8dp"
-                    android:layout_marginBottom="16dp"
-                    android:orientation="vertical">
-
-                    <Spinner
-                        android:id="@+id/alternate_urls_spinner"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="8dp"
-                        android:dropDownWidth="match_parent"
-                        android:padding="8dp"
-                        android:textColor="?android:attr/textColorPrimary"
-                        android:textSize="@dimen/text_size_micro" />
-
-                    <Button
-                        android:id="@+id/subscribeButton"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginStart="8dp"
-                        android:layout_marginTop="16dp"
-                        android:layout_marginEnd="8dp"
-                        android:focusable="false"
-                        android:text="@string/subscribe_label" />
-
-                    <CheckBox
-                        android:id="@+id/autoDownloadCheckBox"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_gravity="left"
-                        android:focusable="false"
-                        android:checked="true"
-                        android:text="@string/auto_download_label"
-                        android:visibility="gone"
-                        tools:visibility="visible" />
-
-                    <Button
-                        android:id="@+id/stopPreviewButton"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_margin="8dp"
-                        android:focusable="false"
-                        android:text="@string/stop_preview"
-                        android:visibility="gone"
-                        tools:visibility="visible" />
+                    <ImageButton
+                        android:id="@+id/closeButton"
+                        android:layout_width="@dimen/close_icon_size"
+                        android:layout_height="@dimen/close_icon_size"
+                        android:layout_alignParentRight="true"
+                        android:layout_marginTop="10dp"
+                        android:layout_marginRight="10dp"
+                        android:background="@android:color/transparent"
+                        android:foregroundTint="@color/white"
+                        android:src="@drawable/close_icon"
+                        app:tint="?android:attr/textColorPrimary" />
```

Comment:
No need to tint if the icon already has the color. Also, you tint it in two different colors